components/apimgt/org.wso2.carbon.apimgt.usage/org.wso2.carbon.apimgt.usage.client/src/main/java/org/wso2/carbon/apimgt/usage/client/impl/APIUsageStatisticsRestClientImpl.java

```diff
@@ -725,11 +724,19 @@ public static OMElement buildOMElement(InputStream inputStream) throws Exception
         //extending lucene query with time ranges
         try {
             query = APIUsageStatisticsClientConstants.REQUEST_TIME + ": [" + RestClientUtil.getFloorDateAsLong(fromDate)
-                    + " TO " + RestClientUtil.getCeilingDateAsLong(toDate) + "]";
+                    + " TO " + RestClientUtil.getCeilingDateAsLong(toDate) + ']';
         } catch (ParseException e) {
             handleException("Error occurred while Error parsing date", e);
         }
 
+        //if My APIs stat, add constraint with API publisher
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/c5071cbc7f3c39fed2489c288179e0fe93507544